includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -3520,6 +3520,13 @@ public function filter_my_account_my_orders_actions( $actions, $order ) {
 		if ( is_order_received_page() && in_array( $order->get_payment_method_title(), $methods_with_delayed_confirmation, true ) && $order->has_status( OrderStatus::PENDING ) ) {
 			unset( $actions['pay'], $actions['cancel'] );
 		}
+
+		// If the order has a checkout session ID and is pending, hide the action buttons. The orders with checkout sessions take a while to be processed via webhooks.
+		$checkout_session_id = WC_Stripe_Order_Helper::get_instance()->get_stripe_checkout_session_id( $order );
```

PHPMD (2.15.0)

[error] The variable $checkout_session_id is not named in camelCase. (undefined)

(CamelCaseVariableName)